pkg/sqlite/load.go

```diff
@@ -612,12 +612,39 @@ func (s *sqlLoader) addAPIs(tx *sql.Tx, bundle *registry.Bundle) error {
 	return nil
 }
 
+// checkIfPackageExists is a method responsible for determing
+// whether the @packageName package exists in the package
+// table and returns an error if that package is non-existent.
+func (s *sqlLoader) checkIfPackageExists(packageName string) error {
+	query := `
+	SELECT DISTINCT name
+	FROM package
+	WHERE name=$1
+	`
```

Comment:
It appears that are several tables that all track the various package names (e.g. `channel`, `channel_entry`, etc.) and I ended up going with the `package` table for this information. I'm not entirely sure what table should serve as the source of truth for packages, but this appears to be a sufficient check in most/all cases.